snapdtool/info_file.go

```diff
@@ -35,7 +36,8 @@ import (
 // indicates whether or not the snapd-apparmor binary installed via the
```

Comment:
the doc comment needs adjusting as it mentions still infoPath.

Reply:
done